<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeWindow</class>
 <widget class="QMainWindow" name="ModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mode</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Do you want to continue as an admin or as an user?</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="ListButton">
          <property name="text">
           <string>Adoption List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adminButton">
          <property name="text">
           <string>Admin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="userButton_2">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
